helm_deploy/hmpps-interventions-ui/values.yaml

```diff
@@ -54,17 +54,6 @@ generic-service:
     v0_47_enabled: false
     modsecurity_snippet: |
       SecRuleEngine On
-      SecAuditEngine RelevantOnly
-      SecAuditLog /var/log/nginx/error.log
-      SecAuditLogType Serial
-      SecRequestBodyAccess On
-      SecAuditLogParts ACHKZ
-      SecAuditLogFormat JSON
-      SecAction "id:900110,phase:1,nolog,pass,t:none,setvar:tx.inbound_anomaly_score_threshold=6,setvar:tx.outbound_anomaly_score_threshold=4"
       SecRuleUpdateActionById 949110 "t:none,deny,status:423,logdata:%{SERVER_NAME}"
       SecRuleUpdateActionById 959100 "t:none,deny,status:423,logdata:%{SERVER_NAME}"
-      SecRuleRemoveById 921110
-      SecRuleUpdateTargetById 942100 "!ARGS"  # disable libinjection testing on request args. to disable all SQLi testing, modify rules in the range 942100-942999
-      SecRuleRemoveByTag language-php
-      SecRuleRemoveByTag language-shell
-      SecRuleRemoveByTag platform-windows
+      SecAction "id:900110,phase:1,nolog,pass,t:none,setvar:tx.inbound_anomaly_score_threshold=50,setvar:tx.outbound_anomaly_score_threshold=50"
```

Comment:
main change is here: `inbound_anomaly_score_threshold=50` instead of `inbound_anomaly_score_threshold=6` on line 63